### PR TITLE
Strategy and pedia texts for units

### PR DESCRIPTION
**Pull request by MeuhV1**
Also fixed the English UHV help text (changed Ship of the Line to Man-o-war)
Corrected value of the sewer in the List of Features

Other units/buildings etc. will come in later PR.

Commit 2:
Strategy and pedia texts for vanilla units

### CODE REVIEW

File: Assets/XML/Text/Civilopedia.xml [hunk 1]
```diff
@@ -6255,4 +6271,308 @@ Your stability level can change during a stability check. As long it is higher t
...
+		<German>[TAB]Prussia made early use of the title "fusilier" for various types of infantry. In 1705, the Foot Guards (Leibgarde zu Fuss) were designated as Fusilier Guards. By 1837, low-quality infantry raised from garrison companies also were named fusiliers. These latter units were dressed in blue with low mitre caps. Between 1740 and 1743 on Frederick the Great raised 14 separate Fusilier Regiments (numbers 33-40, 41-43 and 45-48). Except for the mitre caps, these new regiments were identical in appearance, training and role to the existing line infantry (musketeers).[PARAGRAPH:2]Subsequently, Prussia and several other German states used the designation Fusilier to denote a type of light infantry, dressed in green, that acted as skirmishers. In the Prussian Army, they had been formed in 1787 as independent battalions, with many of the Officers having had experience in the American Revolutionary War. The Prussian reforms of 1808 absorbed the fusiliers as the third battalion of each line infantry regiment. Now wearing the same Prussian blue uniforms as standard musketeers, they were distinguished by black leather belts, and a slightly different arrangement of cartridge pouch.[
```
Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: Sounds like this is from the wiki article on modern gendarmerie, the pedia text should focus more on the earlier heavy cavalry meaning of the term (gens d'arme).

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: siege cannons

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: Looks like a duplicate.

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: There's a bunch of non-ASCII characters in here that probably won't be rendered correctly.

File: Assets/XML/Text/Units.xml [hunk 2]
```diff
@@ -1251,19 +1223,19 @@
...
-		<French>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Melee Units[COLOR_REVERT].</French>
-		<German>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Melee Units[COLOR_REVERT].</German>
-		<Italian>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Melee Units[COLOR_REVERT].</Italian>
-		<Spanish>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Melee Units[COLOR_REVERT].</Spanish>
+		<English>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has three extra strength points.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: The Hwacha also has a reduced city attack modifier.

File: Assets/XML/Text/Units.xml [hunk 3]
```diff
@@ -2941,6 +2929,14 @@
 		<Italian>Satellite</Italian>
 		<Spanish>Satellite</Spanish>
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_UNIT_SATELLITE_STRATEGY</Tag>
+		<English>[COLOR_UNIT_TEXT]Satellites[COLOR_REVERT] can see the [COLOR_UNIT_TEXT]Satellite[COLOR_REVERT] units, settle in a city as a [COLOR_HIGHLIGHT_TEXT]Research Satellite[COLOR_REVERT], [COLOR_HIGHLIGHT_TEXT]Commercial Satellite[COLOR_REVERT] or [COLOR_HIGHLIGHT_TEXT]Military Satellite[COLOR_REVERT] specialist, can explore rival territory, are invisible to all units except [COLOR_UNIT_TEXT]Satellite[COLOR_REVERT] units and begin with the [COLOR_HIGHLIGHT_TEXT]Sentry[COLOR_REVERT] promotion.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: can see other Satellite units

File: Assets/XML/Text/Civilopedia.xml (hunk 1 above)
Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: I found a better text.

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: I'm not really sure what you mean by this.

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: What is duplicated? I not sure what you mean by this.

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Fixed.

File: Assets/XML/Text/Units.xml (hunk 2 above)
Comment (MeuhV1) [reply Assets/XML/Text/Units.xml]: Fixed.

File: Assets/XML/Text/Units.xml (hunk 3 above)
Comment (MeuhV1) [reply Assets/XML/Text/Units.xml]: Fixed.

**Comment by MeuhV1** [comment]
Uploaded some more texts. (buildings, wonders and leaders)

I found a better text for the Gendarme.
What do you mean by duplicate? I can't find anything that is duplicated.
What do you mean by "siege cannon"? I can't find what you are referring to.
Fixed the other small issues.

**Comment by dguenms** [comment]
Siege cannon refers to a misspelling, I think it was "seige cannon" before.
The duplicate was about a text key being defined twice with the same text.

File: Assets/XML/Text/Civilopedia.xml [hunk 4]
```diff
@@ -5865,11 +5865,11 @@ Your stability level can change during a stability check. As long it is higher t
...
...anies (or by a combination of these two methods). Captains dreaded a reduction of their company as a diminution of their prestige and income, and worked hard to prevent this&#8212;which companies were reduced usually reflected the influence of the respective captains at court.[PARAGRAPH:2]The gendarme companies were permanently stationed in towns in the provinces throughout France, subject to be summoned during wartime and concentrated in the Royal armies. Some became closely associated with the towns where they were stationed. If these garrison towns did not have sufficient resources to support the gendarmes present, as was often the case, individuals often found lodging in nearby areas. This lack of lodging could apply even in times of peace, when many of the men retired to their homes instead of remaining in the garrison (particularly in winter), and despite the contemporary system of giving leave, which allowed up to one quarter of the company to be away at any given time. Men who were away for these reasons were to be brought back to the company by the captains when ordered to do so by the provincial governor. Long-term absence was a chronic problem in the companies.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: `&#8212;` not sure what this is but I don't think it's supposed to be there. I have also seen -- to separate lines in this text.

File: Assets/XML/Text/Civilopedia.xml [hunk 5]
```diff
@@ -6169,10 +6177,570 @@ Your stability level can change during a stability check. As long it is higher t
...
...their existing water supplies.[PARAGRAPH:2]The Aqua Appia was the first test of Roman engineering of its type and is quite unsophisticated in comparison to Rome's ten other aqueducts. Raffaelo Fabretti, one of the first to excavate the Appia, characterises it as "The first fruits of Rome's Foresight and greatness". Nevertheless, the Appia was kept in use into the era of Augustus Caesar through regular maintenance, renovations and even an expansion. Little of the original material remains today and much of information on the aqueduct comes from Frontinus who was appointed as water commissioner and recorded the technical, and some historical, details of the Appia about one hundred years after its completion.[PARAGRAPH:2]Although there were no formal Roman predecessors for the Aqua Appia, there were plenty of examples that existed in the Greek and Etruscan world. Greek influence came from their use of terracota pipes that were laid along the bottom of a channel or a large tunnel. This technique relates to the Roman technique in that it uses a water channel within a larger tunnel. The difference between the two is the fact that the Romans began to favour masonry conduits rather than th
```
Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: `&amp;` looks wrong.

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: Would be good to standardise the spelling of Gondeshapur.

**Comment by MeuhV1** [comment]
I found the errors and fixed them.

**Comment by MeuhV1** [comment]
Added project pedia texts. I gave the Mission to Mars and the mars components the same pedia text, as I didn't find good texts for each individual component. So I gave all of them a generic text about Mars colonization.

File: Assets/XML/Text/Civilopedia.xml (hunk 5 above)
Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: `&#8220;`shouldn't be here.

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: I assume these are Greek and Cyrillic characters, the game cannot render those.

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: `&#8201;`

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: Literal ü

Comment (dguenms) [reply Assets/XML/Text/Civilopedia.xml]: More unescaped characters in other languages.

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: `&#8364;`

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: `&amp;`

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: shaivite -> shivaite

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: Remove "(see Minnesang)"

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: I think there should be more text that includes background for the mine and its history before 1950.

Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: `&#8212;`

File: Assets/XML/Text/Units.xml [hunk 6]
```diff
@@ -1095,11 +1095,11 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_KOREAN_HWACHA_STRATEGY</Tag>
-		<English>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has three extra strength points.</English>
-		<French>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has three extra strength points.</French>
-		<German>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has three extra strength points.</German>
-		<Italian>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has three extra strength points.</Italian>
-		<Spanish>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has three extra strength points.</Spanish>
+		<English>The [COLOR_UNIT_TEXT]Hwacha[COLOR_REVERT], the Korean replacement for the [COLOR_UNIT_TEXT]Catapult[COLOR_REVERT], has three extra strength points and has an reducde attack bonus when attacking cities.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: reducde -> reduced

**Review by dguenms — CHANGES_REQUESTED** [review]

File: Assets/XML/Text/Civilopedia.xml (hunk 5 above)
Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: It is the ampersand (&) sign. The game renders those correctly. Do you want me to keep them or replace them with the word "and"?

File: Assets/XML/Text/Civilopedia.xml (hunk 4 above)
Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: It is some kind of dash, commonly used in pedia texts. I thought I replaced them with regular hyphens, but I only replaced the &#8111; (another type of dash) with hyphens.

The -- is a mistake I didn't catch myself.

File: Assets/XML/Text/Civilopedia.xml (hunk 5 above)
Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Done

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: True. Somehow I missed them.

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Fixed

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: I copied the text of the Neuschwanstein castle from Civ 5. I assumed that those texts were already formatted to XML format, but apperantly they are not.

Fixed the issues. (For all texts from Civ 5)

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Fixed

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Fixed

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Same as above

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Fixed

File: Assets/XML/Text/Units.xml (hunk 6 above)
Comment (MeuhV1) [reply Assets/XML/Text/Units.xml]: Fixed

File: Assets/XML/Text/Civilopedia.xml (hunk 5 above)
Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: I included a small paragraph about the mine. I couldn't find more information about the history before 1950.

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Fixed

Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: Fixed. I also fixed the same issue for some other (already existing) texts.

**Comment by MeuhV1** [comment]
Fixed all comments, except &#amp;. That is the ampersand (&) sign and renders normal ingame. I can change those to the word "and" if you want.

File: Assets/XML/Text/Civilopedia.xml (hunk 5 above)
Comment (dguenms) [reply Assets/XML/Text/Civilopedia.xml]: Does a literal ampersand not work?

File: Assets/XML/Text/Civilopedia.xml [hunk 7]
```diff
@@ -14,8 +14,8 @@
...
...lianti e l'Impero fu costretto a contrarre enormi prestiti per pagare i propri debiti: l'inizio di un circolo vizioso che non port� che a debiti sempre maggiori per l'Impero.[PARAGRAPH:2]Anche se furono intraprese delle serie riforme per modernizzare l'impero, i costumi e le tradizioni impedirono ai riformatori di agire con la necessaria celerit&#224;. Inoltre la crescente ondata di nazionalismo cominci� a dilaniare lo stato: Serbi, Bulgari, Rumeni e Greci si dichiararono tutti indipendenti e fondarono i rispettivi stati. All'epoca dello scoppio della Prima Guerra Mondiale, l'Impero ottomano aveva perduto quasi tutti i suoi possedimenti in Europa. Nel disperato tentativo di conservare il poco rimasto, gli Ottomani si allearono con le potenze centrali (Germania e Austria), solo per essere sconfitti dalla rivolta araba del Vicino Oriente guidata dai Britannici. Al termine della guerra i vincitori si ripartirono le spoglie dell'Impero ottomano alla conferenza di pace di Parigi, anche se la Rivoluzione Turca imped� la spartizione anche dell'Anatolia. L'Impero ottomano conobbe la fine nel 1922 con la fondazione dello stato turco e la deposizione dell'ultimo sultano, Mehmet VI.</Italian>
```
Comment (dguenms) [inline Assets/XML/Text/Civilopedia.xml]: I see a lot of � in the Ottoman Empire text

**Review by dguenms — CHANGES_REQUESTED** [review]

File: Assets/XML/Text/Civilopedia.xml (hunk 5 above)
Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: It doesn't. It will result in an error.

File: Assets/XML/Text/Civilopedia.xml (hunk 7 above)
Comment (MeuhV1) [reply Assets/XML/Text/Civilopedia.xml]: IIRC that was copied from the original Warlords text. I have no idea why it got corrupted. Anyway, I recopied the original Warlords text and it should be fixed now.

**Review by dguenms — APPROVED** [review]

**Pull request merged**